### DISCUSSION TITLE
registrierte Instanzen

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3586] [General]
hallo Georg,
ist es möglich unter Einstellungen, Kontoübersicht nicht mehr gültige Instanzen zu löschen, bzw. löschen zu lassen ?
Habe durch viel Probiererei und Tests deren zu viel in meinem Konto.
Diese zwei sind nicht mehr aktiv
![grafik](https://github.com/user-attachments/assets/2e1b3853-d913-4815-a7af-0d099dd8983b)

**Comment by ledermann** [comment]
Ein manuelles Löschen ist hier nicht vorgesehen. Nicht mehr verwendete Instanzen verschwinden aber automatisch 7 Tage nach der letzten Verwendung.
